metadata.json

```diff
@@ -103,11 +103,11 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": ">= 3.7.0 < 2015.4.0"
+      "version_requirement": ">= 2015.4.0 < 2016.4.0"
```

Comment:
Actually.. the whole "pe" block of the requirements is deprecated. We can remove that section and keep just the "puppet" section.